src/System.Runtime/tests/System/HandleTests.cs

```diff
@@ -30,10 +32,21 @@ public static void  RuntimeMethodHandleTest()
         Assert.True(Co6006LateBoundDelegate.iInvokeCount == 1);
     }
 
+    [Fact]
+    public static void  GenericMethodRuntimeMethodHandleTest()
+    {
+        // Make sure uninstantiated generic method has a valid handle
+        MethodInfo mi1 = typeof(Base).GetMethod("GenericMethod");
+        MethodInfo mi2 = (MethodInfo)MethodBase.GetMethodFromHandle(mi1.MethodHandle);
+        Assert.Equal(mi1, mi2);
```

Comment:
Referential equality is not guaranteed for MethodInfos. On CLR it works most of the time because of caching, but on CoreRT the reflection API will pretty much always make a new object instance.